Solutions/Marain.Workflow.Abstractions/Marain.Workflow.Abstractions.csproj

```diff
@@ -1,24 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\Common.NetStandard_2_0.proj" />
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup>
-    <PackageLicenseExpression>AGPL-3.0-or-later</PackageLicenseExpression>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
Have we conciously changed this from AGPL to Apache? If so what was the reasoning behind that? (Marain is, in general , AGPL restricting it to non-commercial/compatible OSS use.)

Reply:
Having just done a quick check, we have plenty of examples of Apache-2.0 across Marain, Menes and Corvus.

@HowardvanRooijen - can you comment on this please?

Reply:
Apache 2.0 for any low value frameworks. AGPL for anything with commercial value.

Reply:
Updated everything to AGPL.